proxy/http/HttpTunnel.cc

```diff
@@ -589,8 +589,8 @@ HttpTunnel::kill_tunnel()
     ink_assert(producer.alive == false);
   }
   active = false;
-  this->deallocate_buffers();
   this->deallocate_redirect_postdata_buffers();
+  this->deallocate_buffers();
```

Comment:
Yes, @scw00 has done it in the last commit. The comment is only for the first commit.